scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -397,6 +397,9 @@ public static void PublishSharedFramework(BuildTargetContext c, string outputDir
             File.Copy(
                 Path.Combine(Dirs.Corehost, HostPolicyBaseName),
                 Path.Combine(SharedFrameworkNameAndVersionRoot, HostPolicyBaseName), true);
+            File.Copy(
+                Path.Combine(Dirs.Corehost, DotnetHostFxrBaseName),
+                Path.Combine(SharedFrameworkNameAndVersionRoot, DotnetHostFxrBaseName), true);
 
```

Comment:
There is the host executables in Constants which is an array of host dependencies.